vespalib/src/tests/stllike/lookup_benchmark.cpp

```diff
@@ -95,6 +95,41 @@ size_t benchHashMapVespaLib2(size_t sz, size_t numLookups)
     return benchM(set, sz, numLookups);
 }
 
+std::unique_ptr<char []> createData(size_t sz) {
+    auto data = std::make_unique<char []>(sz);
+    for (size_t i(0); i < sz; i++) {
+        data.get()[i] = i + '0';
+    }
+    return data;
+}
+
+size_t benchXXHash32(size_t sz, size_t numLookups) {
+    auto data = createData(sz);
+    size_t sum(0);
+    for (size_t i(0); i < numLookups; i++) {
+        sum += XXH32(data.get(), sz, 0);
+    }
+    return sum;
+}
+
+size_t benchXXHash64(size_t sz, size_t numLookups) {
+    auto data = createData(sz);
+    size_t sum(0);
+    for (size_t i(0); i < numLookups; i++) {
+        sum += XXH64(data.get(), sz, 0);
+    }
+    return sum;
+}
+
+size_t benchLegacyHash(size_t sz, size_t numLookups) {
+    auto data = createData(sz);
+    size_t sum(0);
+    for (size_t i(0); i < numLookups; i++) {
+        sum += vespalib::hashValue(data.get(), sz);
+    }
+    return sum;
+}
```

Comment:
Since all these benchmarks are the same aside from the evaluation call, could consider factoring out as template argument to a generic benchmark loop function.

Reply:
Yes, I will make one more round.